src/import_edf.jl

```diff
@@ -331,45 +608,37 @@ in `\$path/samples/`, and write the Onda signals and annotations tables to
 "edf", and if a prefix is provided for signals but not annotations both will use
 the signals prefix.  The prefixes cannot reference (sub)directories.
 
-Returns `(; recording_uuid, signals, annotations, signals_path, annotations_path)`.
-
-Samples are extracted with [`edf_to_onda_samples`](@ref), and EDF+ annotations are
-extracted with [`edf_to_onda_annotations`](@ref) if `import_annotations==true`
-(the default).
+Returns `(; recording_uuid, signals, annotations, signals_path, annotations_path, plan)`.
 
-Collections of `EDF.Signal`s are mapped as channels to `Onda.Signal`s via simple
-"extractor" callbacks of the form:
+This is a convenience function that first formulates an import plan via
+[`plan_edf_to_onda_samples`](@ref), and then immediately executes this plan with
+[`edf_to_onda_samples`](@ref).
 
-    edf::EDF.File -> (samples_info::Onda.SamplesInfo,
-                      edf_signals::Vector{EDF.Signal})
+The samples and executed plan are returned; it is **strongly advised** that you
+review the plan for un-extracted signals (where `:kind` or `:channel` is
+`missing`) and errors (non-`nothing` values in `:error`).
 
-`store_edf_as_onda` automatically uses a variety of default extractors derived
-from the EDF standard texts; see `src/standards.jl` and
-[`extract_channels_by_label`](@ref) for details. The caller can provide
-alternative extractors via the `custom_extractors` keyword argument, and the
-[`edf_signals_to_samplesinfo`](@ref) utility can be used to extract a common
-`Onda.SamplesInfo` from a collection of EDF.Signals.
+Groups of `EDF.Signal`s are mapped as channels to `Onda.Samples` via
+[`plan_edf_to_onda_samples`](@ref).  The caller of this function can control the
+plan via the `labels`, `units`, and `preprocess_labels` keyword arguments, all
+of which are forwarded to [`plan_edf_to_onda_samples`](@ref).
 
 `EDF.Signal` labels that are converted into Onda channel names undergo the
 following transformations:
 
-- the label's prepended signal type is matched against known types, if present
-- the remainder of the label is whitespace-stripped, parens-stripped, and lowercased
+- the label is whitespace-stripped, parens-stripped, and lowercased
 - trailing generic EDF references (e.g. "ref", "ref2", etc.) are dropped
 - any instance of `+` is replaced with `_plus_` and `/` with `_over_`
 - all component names are converted to their "canonical names" when possible
-  (e.g. for an EOG matched channel, "eogl", "loc", "lefteye", etc. are converted
-  to "left").
-
-`EDF.Signal`s which get extracted into more than one `Onda.Samples` are removed
-and an `AmbiguousChannelError` displayed as a warning.
+  (e.g. "m1" in an EEG-matched channel name will be converted to "a1").
```

Comment:
Is that true? I thought we explicitly removed that conversion a while back.

Reply:
if we didn't, we should, because strictly speaking M1 is the left mastoid and A1 is the left earlobe (even though a lot of labs used them interchangeably)

Reply:
I'm pretty sure we did remove that, I'll update teh docstring